help/pt_BR/config_comments.yml

```diff
@@ -1,47 +1,57 @@
-        camera_editor_f1_tooltip: Mostrar a dica no canto inferior esquerdo, que explica como retornar a GUI
+    editor:
+        sync: Permite sincronizar a posição da câmera no editor de câmera, que visualiza o perfil atual da câmera
+        loop: Repete a reprodução do suporte selecionado atualmente
+        overlay: Ativa a sobreposição do editor de câmera
+        overlay_rl: Caminho do local do recurso para a textura que será usada como sobreposição
+        f1_tooltip: Mostrar a dica no canto inferior esquerdo, que explica como retornar a GUI
+        position: Mostrar propriedades da câmera quando o modo de sincronização está ativado
+        rule_of_thirds: Mostrar guias de regra de terços
+        crosshair: Mostra uma mira no meio da tela do editor de câmera
+        letter_box: Mostrar caixa postal (também conhecidas como barras pretas) que estreitam ou ampliam a foto da câmera como referência
+        aspect_ratio: Relação de aspecto para o recurso de caixa postal do editor de câmera
+        
+    flight:
+        forward: Mover a câmera para frente
+        backward: Mover a câmera para trás
+        left: Mover a câmera para esquerda
+        right: Mover a câmera para direito
+        up: Subir a câmera
+        down: Descer a câmera
+        camera_up: Inclinar a câmera para cima em direção ao céu
+        camera_down: Inclinar a câmera para baixo em direção ao chão
+        camera_left: Deslocar a câmera para a esquerda
+        camera_right: Deslocar a câmera para a direita
+        fov_minus: Diminuir o campo de visão
+        fov_plus: Aumentar o campo de visão
+        roll_minus: Rolar no sentido anti-horário
+        roll_plus: Rolar no sentido horário
+        speed_minus: Diminuir o fator de velocidade geral (em vez de girar o mouse para baixo)
+        speed_plus: Aumentar o fator de velocidade geral (em vez de girar o mouse para cima)
 
     smooth:
-        camera_smooth_clamp: Prender da afinação suave da câmera entre -90 e 90 graus?
-
-        roll_speed: Velocidade de aceleração do rolamento
-        roll_friction: Atrito de aceleração do rolamento (com que rapidez diminui)
-
-        fov_speed: Velocidade de aceleração FOV
-        fov_friction: Atrito de aceleração FOV (com que rapidez diminui)
-
-        smooth_enabled: Ativar câmera suave.
-        mouse_x_friction: Atrito suave do mouse X
-        mouse_y_friction: Atrito suave do mouse Y
+        enabled: Ativar câmera suave
+        x_friction: Deslizamento suave do mouse em X
+        y_friction: Deslizamento suave do mouse em Y
+        clamp: Prender a câmera numa inclinação suave entre -90 e 90 graus de intervalo?
+        roll_speed: Velocidade de aceleração de rolagem
+        roll_friction: Deslizamento de aceleração de rolagem (com que rapidez diminui)
+        fov_speed: Velocidade de aceleração do FOV
+        fov_friction: Delizamento de aceleração do FOV (com que rapidez diminui)
```

Comment:
Here `minema` section [is missing](https://github.com/mchorse/aperture/compare/1.3.6...1.12#diff-01caf0c8a890718ab1c240aed7357a4aR59) too.